### PR TITLE
slmora-feature-a - update - SLMORA - 202005101312

### PR DESCRIPTION
**Pull request by SLMORA**
New changes from the SLMORA

### CODE REVIEW

File: src/main/webapp/mora_page1.jsp [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Mora Page 1 Title</title>
+</head>
+<body>
+    <h2>Hello World!</h2>
+	<h3>Line added morapage1.jsp 001</h3>
+	<h3>Line added morapage1.jsp 002</h3>
+	<h3>Line added morapage1.jsp 003</h3>
```
Comment (UmeshGunasekara) [inline src/main/webapp/mora_page1.jsp:16]: Change the **names**  
```suggestion
	<h3>Line added MoraPage1.jsp 001</h3>
	<h3>Line added MoraPage1.jsp 002</h3>
	<h3>Line added MoraPage1.jsp 003</h3>
```

File: src/main/webapp/mora_page1.jsp [hunk 2]
```diff
@@ -0,0 +1,19 @@
+<%--
+  Created by IntelliJ IDEA.
+  User: umesh
```
Comment (UmeshGunasekara) [inline src/main/webapp/mora_page1.jsp]: Change this **Created Section**
```suggestion
  User: umesh
```

**Review by UmeshGunasekara — CHANGES_REQUESTED** [review]

File: src/main/webapp/mora_page1.jsp (hunk 1 above)
Comment (UmeshGunasekara) [reply src/main/webapp/mora_page1.jsp:16]: OK I'll re do and submit

Comment (SLMORA) [reply src/main/webapp/mora_page1.jsp:16]: Got It

File: src/main/webapp/mora_page1.jsp (hunk 2 above)
Comment (SLMORA) [reply src/main/webapp/mora_page1.jsp]: I'll Do

Comment (SLMORA) [reply src/main/webapp/mora_page1.jsp]: Done

Comment (UmeshGunasekara) [reply src/main/webapp/mora_page1.jsp]: Thank you

**Review by UmeshGunasekara — APPROVED** [review]

File: src/main/webapp/mora_page1.jsp [hunk 3]
```diff
@@ -0,0 +1,18 @@
+<%--
+  User: umesh
+  Date: 05/10/2020
+  Time: 1:10 PM
+  To change this template use File | Settings | File Templates.
+--%>
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Mora Page 1 Title</title>
```
Comment (UmeshGunasekara) [inline src/main/webapp/mora_page1.jsp]: Change this
```suggestion
    <title>Mora Title</title>
```

**Review by UmeshGunasekara — CHANGES_REQUESTED** [review]

File: src/main/webapp/mora_page1.jsp (hunk 3 above)
Comment (SLMORA) [reply src/main/webapp/mora_page1.jsp]: Done

**Review by UmeshGunasekara — APPROVED** [review]

**Pull request merged**